CHANGELOG.md

```diff
@@ -24,11 +25,23 @@ and includes an additional section for migration notes.
 - *ORCA-597*
   - Server access logging is now enabled for graphql application load balancer.
 
+### Changed
+- *ORCA-573* Updated ORCA DB user password to now have a stronger password requirement. See migration notes for details.
+
+
```

Comment:
We typically only have one line of whitespace between sections.

Reply:
fixed